packages/css/src/components/info-button/info-button.template.ts

```diff
@@ -3,15 +3,15 @@ import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map';
 import { ifDefined } from 'lit-html/directives/if-defined';
 
-export function infoButtonTemplate({ active, label, onClick }: InfoButton) {
+export function infoButtonTemplate({ active, label, onClick, iconOnly }: InfoButton) {
```

Comment:
Waar is deze change voor nodig? Anders dan 'n Storybook Control zie ik 'm nergens ingezet worden. Als ie écht nodig is, waarom is het niet nodig in Fractal?

Reply:
Hij _was_ nodig omdat Bram expliciet de `InfoButton` had gebruikt in zijn ontwerpen, met een label. Maar de ontwikkelingen op dat niveau zijn al een ander pad in geslagen, en wordt een ticket ergens in een later traject. Voor nu haal ik deze changes weer weg.